src/schema/v2/conversation/index.ts

```diff
@@ -329,6 +329,13 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ from_last_viewed_message_id }) => from_last_viewed_message_id,
     },
+    toLastViewedMessageID: {
+      type: GraphQLString,
+      resolve: ({ from_last_viewed_message_id, ...rest }) => {
+        console.log("### rest", rest)
+        return from_last_viewed_message_id
```

Comment:
This might be pretty confusing long-term. Why is the field name different from the field as returned from the API? Shouldn't it be `fromLastViewedMessageID`? If we're going to change it to the inverse of the data being returned there has to be a really good reason 😄

Reply:
Apologies 🙇 
I forgot to clean up this new field https://github.com/artsy/metaphysics/pull/4952/commits/1a003df7ec83ebed5a6b325324ff35f503354622 should be okay in this commit 😓